nb2workflow/deploy.py

```diff
@@ -298,15 +308,40 @@ def _build_with_kaniko(git_origin,
         return container_metadata
 
 
-def _build_with_docker(git_origin, 
+def _extract_resource_requirements(local_repo_path, ontology_URL=default_ontology_URL):
+    sp.check_call([
+        "wget", ontology_URL, "-O", "ontology.ttl"
```

Comment:
why don't you use `requests.get`?

Reply:
The code was removed, since now we pass ontology_path parameter to Ontology directly as it was suggested by @dsavchenko . However there is another place in conftest.py where I was using wget for the same purpose. This code was modified to use requests API